### PR TITLE
https://github.com/OpenSCAP/scap-workbench/issues/28

### PR DESCRIPTION
**Pull request by ybznek**
screenshot_2015-08-14_18-53-28

The pullrequest:
 - Add scrollbar
 - Make benchmark names consistent
 - Divide benchmark into two "groups" preffered/non-preffered

### CODE REVIEW

**Comment by mpreisler** [comment]
Hi,
thanks for your contribution! Please try to match the code style of the project. Curly braces go on a new line.

Other than that I see no major issues.

Could you post a screenshot of how this looks like?

**Comment by mpreisler** [comment]
#28

File: src/SSGIntegrationDialog.cpp [hunk 1]
```diff
@@ -90,15 +90,33 @@ void SSGIntegrationDialog::scrapeSSGVariants()
...
+
+        else
+            label[0] = label[0].toUpper(); // Capitalize first letter
+
+        QPushButton* button = new QPushButton(label, mUI.content);
         button->setProperty("ssg_variant", QVariant(name));
-        mUI.variants->layout()->addWidget(button);
+
+        if (favorite)
+            mUI.variantsLayout->insertWidget(1, button); // insert button before text and divider
```
Comment (mpreisler) [inline src/SSGIntegrationDialog.cpp]: Does this mean that favorites end up being in reverse-sorted order and everything else is in sorted order?

Comment (ybznek) [inline src/SSGIntegrationDialog.cpp]: Yes... I understand favorites as some SET of preferred options. I don't know WHICH items should be preferred and HOW we want to order them.
Current implementation is the easiest possible.

Comment (mpreisler) [inline src/SSGIntegrationDialog.cpp]: In my opinion it looks odd that the order is reversed. How about remembering the last index of the favorites set and always appending?

**Comment by mpreisler** [comment]
Could you please add `Fedora` to favorites?

**Comment by mpreisler** [comment]
Thx

**Pull request merged**